src/main/java/core/basesyntax/Main.java

```diff
@@ -0,0 +1,18 @@
+package core.basesyntax;
+
+public class Main {
+    public static void main(String[] args) {
+        ColorSupplier colorSupplier = new ColorSupplier();
+        FigureSupplier figureSupplier = new FigureSupplier(colorSupplier);
+
+        Figure[] figures = new Figure[6];
+        for (int i = 0; i < figures.length; i++) {
+            figures[i] = figureSupplier.getRandomFigure();
+        }
+        figures[5] = new IsoscelesTrapezoid(colorSupplier.getRandomColor(), 10, 20, 15);
+
+        for (Figure figure : figures) {
+            figure.draw();
+        }

```

Comment:
use only one loop
why do you create trapezoid here? Here's what you should do in Main:
![image](https://github.com/mate-academy/jv-oop-advanced/assets/49767049/dbf16388-22d0-4b68-b910-8df1d621c824)